lib/rex/parser/arguments.rb

```diff
@@ -70,10 +104,21 @@ def parse(args, &_block)
       def usage
         txt = ["\nOPTIONS:\n"]
 
-        fmt.sort.each do |entry|
+        fmt.each do |entry|
```

Comment:
It probably makes sense to keep these sorted. The rationale being, developers will normally put new flags in arbitrary places - normally the last place in an array, or in some scenarios flags will be computed dynamically - and the random order will be confusing to the user:

https://github.com/rapid7/metasploit-framework/blob/d4b478ab7179f802f2291c74d28021426b428108/lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb#L44-L54

https://github.com/rapid7/metasploit-framework/blob/master/lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb#L25-L42